README.md

```diff
@@ -210,3 +210,4 @@
 - [`useScreenType`](https://github.com/pankod/react-hooks-screen-type) Determining screen size type for Bootstrap 4 grid.
 - [`useScrollSpy`](https://github.com/Purii/react-use-scrollspy) React hook to automatically update navigation based on scroll position.
 - [`useValueAfter`](https://github.com/bboydflo/use-value-after) Very simple React hook to easily provide different props to a component (comes in handy for testing edge cases)
+- [`use-on-click-outside`](https://github.com/wellyshen/react-cool-onclickoutside) 🖱 React hook to listen for clicks outside of the component(s).
```

Comment:
Is not sorted alphabetically. Also, according to the repo name and docs (https://github.com/wellyshen/react-cool-onclickoutside) the package name should be named `react-cool-onclickoutside`, right?

Reply:
@aarongarciah I have updated it, plz check again.